broker-cli/commands/wallet.js

```diff
@@ -125,8 +126,13 @@ async function newDepositAddress (args, opts, logger) {
 async function commit (args, opts, logger) {
   const { symbol, amount } = args
   const { rpcAddress = null, market } = opts
+  const currentCurrencyConfig = currencyConfig.find(({ symbol: configSymbol }) => configSymbol === symbol)
```

Comment:
I think we've had this conversation before, but why not just have the config inside of the daemon instead of the CLI? What's the drawback?

Reply:
@treygriffith I would rather have it inside of the daemon. I think we have 2 separate configs because we wanted to ship the CLI as a separate entity. Do you know if we have a future item to change this?

Reply:
@dannypaz but I suppose that means either: having the daemon expose an endpoint with this config, or not having the user feedback (e.g. "are you ok committing the max (which is XYZ BTC)?"). I don't want the CLI to reach into the daemon code to get the config.

I couldn't find a ticket for this issue.